src/hostapi/asio/pa_asio.cpp

```diff
@@ -3199,6 +3223,33 @@ static void sampleRateChanged(ASIOSampleRate sRate)
     PA_DEBUG( ("sampleRateChanged : %d \n", sRate));
 }
 
+static void (*bufferSizeChangeCallback)(long newBufferSize);
```

Comment:
According to Naming conventions:
https://github.com/PortAudio/portaudio/wiki/ImplementationStyleGuidelines

> All static variables end with _